### PR TITLE
Integrationdb

### PR DESCRIPTION
**Pull request by neeleshvarma**
please review & merge to master

### CODE REVIEW

File: registration-service/src/main/java/com/egiants/rlm/externalmodel/User.java [hunk 1]
```diff
@@ -13,48 +20,94 @@
...
+		this.role = role;
+		this.createdBy = createdBy;
+		this.lastModifiedBy = lastModifiedBy;
+		this.createdDate = createdDate;
+		this.lastModifiedDate = lastModifiedDate;
+	}
+
+
+	@DynamoDBHashKey(attributeName = "Id")
+	@DynamoDBAutoGeneratedKey
```
Comment (Egiantsgit) [inline registration-service/src/main/java/com/egiants/rlm/externalmodel/User.java]: why you need this ID, in my case email is my primary key

File: registration-service/pom.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.0.2.RELEASE</version>
+		<version>1.5.9.RELEASE</version>
```
Comment (Egiantsgit) [inline registration-service/pom.xml]: why we need to down grade version in this case

Comment (neeleshvarma) [reply registration-service/pom.xml]: I & Mahesh were checking with the compatible versions, when we were having issues to run the program successfully, So we used the version which we used in the previous project which is running successfully for a safe side.

File: registration-service/src/main/java/com/egiants/rlm/externalmodel/User.java (hunk 1 above)
Comment (Egiantsgit) [reply registration-service/src/main/java/com/egiants/rlm/externalmodel/User.java]: I want to generate unique ID by java and it should be in service layer 
In my case I  want to call multiple collections and I use UUID.. not some thing generate by ORM

Comment (neeleshvarma) [reply registration-service/src/main/java/com/egiants/rlm/externalmodel/User.java]: Okay. I can remove id, if you want me to

**Pull request closed without merging**